pmd-java/src/main/java/net/sourceforge/pmd/lang/java/ast/ASTEnumDeclaration.java

```diff
@@ -18,4 +21,29 @@ public Object jjtAccept(JavaParserVisitor visitor, Object data) {
         return visitor.visit(this, data);
     }
 
+    public boolean isNested() {
```

Comment:
I think the `makeOf` methods should be package private anyway so they wouldn't clutter the API. Maybe they could be renamed too, from eg `makeOperationOf` to `makeOfOperation`, I think it's clearer